src/vaillant_netatmo_api/thermostat.py

```diff
@@ -373,6 +373,7 @@ def __init__(
         setpoint_manual: dict = {},
         therm_program_list: list[dict] = [],
         measured: dict = {},
+        boiler_status: bool | None = None,
```

Comment:
I suggest dropping the `None` fallback and using `False` as a default value. Otherwise users of the lib have to do null checks for no reason, as API always returns this value.

```suggestion
        boiler_status: bool = False,
```